package.json

```
@@ -10,15 +10,19 @@
   ],
   "types": "dist/index.d.ts",
   "scripts": {
+    "all": "run-s clean lint-check build test",
```

To fix the formatting, if we run the script before commit
```suggestion
    "all": "run-s clean lint lint-check build test",
```